test/blackbox-tests/test-cases/pkg/opam-package-with-extra-source.t

```diff
@@ -30,21 +30,7 @@ Make a package with an extra-source field and multiple checksums
   > }
   > EOF
 
-  $ opam_repo=$mock_packages
```

Comment:
how come this was deleted?

Reply:
Because all of this gets done when we do `mkpkg`.